.codespell.toml

```diff
@@ -0,0 +1,5 @@
+[codespell]
+skip = '_build,_static,./venv,Makefile,*.sphinx,*.py,*.dot,*.inv,*.js'
```

Comment:
Can these options be specified using normal TOML lists or do they have to use comma separated strings?

Reply:
no, TOML list doesn't seem to work so I've changed to use `.codespellrc` instead